### DISCUSSION TITLE
Custom filter

### DISCUSSION THREAD

**Discussion by breakingbad45** [pocketbase/pocketbase#3674] [Q&A]
can we use limit in pocketbase.like if i want to fetch only last 4 data?

**Comment by ganigeorgiev** [comment]
It depends what do you understand by _"fetch only last 4 data"_.

To limit the returned results with the SDKs, you can use the `getList` method:
```js
const result = await pb.collection("example").getList(1, 4, {
  sort: "-created"
})
```

If you want to prevent users from fetching more than 4 items, you can create a helper [View Collection](https://pocketbase.io/docs/collections/#view-collection), with a query like:

```sql
select id, field1, field2, ...
from yourCollection
order by created desc
limit 4
```
